index.bs

```diff
@@ -13244,6 +13264,162 @@ the exact steps to take if <dfn>an exception was thrown</dfn>, or by explicitly
 </div>
 
 
+<h3 id="synthetic-module-records">Synthetic module records</h3>
+
+A <dfn export>Synthetic Module Record</dfn> is used to represent information about a module that is
+defined by specifications.
+Its exports are derived from a pair of lists, of string keys and of ECMAScript values.
```

Comment:
This sentence seems wrong; it must have been from an earlier draft I never uploaded.